docs/source/courses/intro-software-dev.md

```diff
@@ -11,49 +11,27 @@ on your own machine. No experience is necessary, all you need is a laptop you ca
 The second day of this course will provide some further Python training, and particularly introduce good practices that 
 will pay dividends as you dive deeper into programming. 
 
-Additionally, we will introduce the Linux command line, and 
-version control with Git, key tools for anyone working with data and code.
 
 ## Contents
```

Comment:
If we want this table of contents to match the current slides, we would need the sections to be:

```
**Day 1**

Part 1 
* Aims
* Why learn Python?
* How to work with Python?
* Basics of programming (1/2)
    - Variables
    - Data types

Part 2 
* Basics of programming (2/2)
    - Loops
    - Conditional statements
    - List comprehensions
* Writing your first Python script
* Loading and saving data


**Day 2**

Part 1
* Recap and Q&A
* Using third party libraries
* Functions and methods
* Classes and objects

Part 2
* Errors and exceptions
* Organising your code
* Documenting your code
```

Reply:
I thought we didn't necessarily have to match the slides exactly, the part 1/2 distinction doesn't make as much sense since we're scheduled for continuous chunks for both day 1 and day 2. I'm flexible though!

Reply:
ah yes sounds good! 

We can remove part 1 and 2 split, but maybe keep the same titles as the slides? Some of the ones in the list we don't cover anymore (e.g. "Installing python" which is currently under day 1, or "Editing Python files (IDE, text editor etc.)" which is maybe a duplicate of "Writing your first Python script")